packages/terra-time-input/src/TimePipe.js

```diff
@@ -0,0 +1,33 @@
+function TimePipe(timeFormat) {
+  return function processConformedValue(conformedValue) {
+    if (!timeFormat || timeFormat.length < 0) {
+      return false;
```

Comment:
We should return null or undefined as opposed to false for this function. If feels like you can get into maintenance issues if you allow for different return types from a function